lib/kafee/consumer/brod_monitor.ex

```diff
@@ -0,0 +1,147 @@
+defmodule Kafee.Consumer.BrodMonitor do
+  @moduledoc """
+  Utility module for monitoring Broadway - covering consumer lags
```

Comment:
```suggestion
  Utility module for monitoring consumer lag
```

Being that this works with N adapters, we can keep it generic.